controllers/news.controllers.js

```diff
@@ -15,4 +16,9 @@ exports.sendArticle = (req, res, next) =>{
     const {article_id} = req.params
    selectArticleById(article_id).then((article)=> res.status(200).send({article}))
    .catch(next)
+}
+
+exports.sendOrderedArticles = (req, res, next) =>{
```

Comment:
I'd maybe look at changing the controller name to getArticles. They are ordered, you're right, however, we can be more explicit in just using the method (get) and saying what data is coming back 👍